etl/test/test_data_management/test_DataManager.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from data_management.DataManager import DataManager
```

Comment:
Missing ROOT_DIR decalration:

import os
ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
os.environ['ROOT_DIR'] = ROOT_DIR

This is not needed for the complete test execution but if you want to execute this test case alone.

Reply:
fixed, testcase now also runs on its own

Reply:
fixed, testcase now also runs on its own